tdesign/src/Layout/PaddedSingleCol/PaddedSingleCol.tsx

```diff
@@ -29,7 +29,7 @@ const PaddedSingleCol = ({
       width: "max(25px, calc((100vw - 784px) / 2))",
     },
     ...extraStyle,
-  };
+  } as SxProps<Theme>;
```

Comment:
The previous way of adding a type was more type-safe (`as` makes TS assertion a bit looser than a type annotation on a variable). What's the reason for lowering type safety here?

Reply:
fixed